package.json

```diff
@@ -24,6 +24,7 @@
     "serve-favicon": "*"
   },
   "devDependencies": {
+    "browserify": "^11.0.1",
```

Comment:
@greybubble change these version numbers to * (asterisk) to match the other dependencies.